From d5571346cc22d6a629a7ed2c4d09df49fb3b60c8 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 19 Nov 2019 13:16:43 +0200
Subject: [PATCH] [SW-0] goya2: ppboot: remove stask/frame pointer init

Calling _start from arcboot_rom is done as calling to a function
from plain C code, so we should not tempere with stack since we will
break arcboot_room stack handling

Change-Id: Ibff0d512f702a53dd47f7155d324fadc8a864b99
---

diff --git a/app/crt0_ppboot.S b/app/crt0_ppboot.S
index c80926f..e5d5fd8 100644
--- a/app/crt0_ppboot.S
+++ b/app/crt0_ppboot.S
@@ -16,9 +16,6 @@
 /* lf2flt: to avoid compiler (buildroot wrapper) warning during linkage */
 .global	lf2flt
 lf2flt:
-
-	mov	sp, __stack_top
-	mov	fp, sp
 #endif
 
 #ifdef __aarch64__
